### ISSUE TITLE
Linux windowing error message

### ISSUE THREAD

**Issue by jkiviluo** [spine-tools/Spine-Toolbox#2934]
Getting this everytime starting Toolbox on Ubuntu 22.04.5 LTS with Wayland Windowing.

qt.qpa.wayland: Creating a popup with a parent, QWidgetWindow(0x63c218d30700, name="MainWindowWindow") which does not match the current topmost grabbing popup, QWidgetWindow(0x63c218518200, name="menuFileWindow") With some shell surface protocols, this is not allowed. The wayland QPA plugin is currently handling it by setting the parent to the topmost grabbing popup. Note, however, that this may cause positioning errors and popups closing unxpectedly. Please fix the transient parent of the popup.

**Comment by ptsavol** [comment]
Does Toolbox still work?

**Comment by jkiviluo** [comment]
Yes. This is maybe just a small annoyance, but Toolbox does have some stability issues on my Ubuntu and this may one source.

**Comment by ptsavol** [comment]
You could try switching from wayland to XORG/X11 but since wayland seems to be the default on newer Ubuntu's we may need a proper fix here.

https://beebom.com/how-switch-between-wayland-xorg-ubuntu/
